lua/lean/infoview.lua

```diff
@@ -254,11 +254,16 @@ function Infoview:__was_closed()
 end
 
 --- Retrieve the contents of the infoview as a table.
-function Infoview:get_lines(start_line, end_line)
+---@param start_line number
+---@param end_line number
+---@param diff boolean @whether to get lines from the diff renderer instead
```

Comment:
I'm not a fan of passing bools to functions. Can you make a separate `get_diff_lines` function, with a helper that both functions call if it indeed is needed?

Reply:
Sounds good, what do you think about moving this to `helpers.lua` as well? Just did a search and looks like we're only using this for testing anyways.

Reply:
So I think for now I've left it here because it's also useful interactively (and later programmatically for anything outside of this that wants to see what's in the infoview until we need a more structured way of passing data out of it), so it just seemed like decent API to expose even if it wasn't used internally. `get_diff_lines` seems less likely to be useful to end-users I suppose but if only for symmetry it seems fine to put here too to me.